database/migrations/20220531100155-update-sites.js

```diff
@@ -0,0 +1,195 @@
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    /**
+     * Add altering commands here.
+     *
+     * Example:
+     * await queryInterface.createTable('users', { id: Sequelize.INTEGER });
+     */
+    const transaction = await queryInterface.sequelize.transaction()
+    try {
+      await queryInterface.dropTable("site_members", { transaction })
+      await queryInterface.dropTable("sites", { transaction })
+
+      await queryInterface.createTable(
+        "sites",
+        {
+          id: {
+            allowNull: false,
+            autoIncrement: true,
+            primaryKey: true,
+            type: Sequelize.BIGINT,
+          },
+          repository_name: {
+            allowNull: false,
+            unique: true,
+            type: Sequelize.TEXT,
+          },
+          created_by: Sequelize.TEXT,
+          created_at: {
+            type: Sequelize.DATE,
+            allowNull: false,
+            defaultValue: Sequelize.fn("NOW"),
+          },
+          updated_at: {
+            type: Sequelize.DATE,
+            allowNull: false,
+            defaultValue: Sequelize.fn("NOW"),
+          },
+          agency: Sequelize.TEXT,
+          site_name: Sequelize.TEXT,
+          contact: Sequelize.TEXT,
+          repository_url: Sequelize.TEXT,
+          hosting_id: Sequelize.TEXT,
+          staging_url: Sequelize.TEXT,
+          production_url: Sequelize.TEXT,
+          live_domain: Sequelize.TEXT,
+          redirect_from: Sequelize.JSONB,
+          uptime_id: Sequelize.TEXT,
+          uptime_url: Sequelize.TEXT,
+          launched_at: Sequelize.DATE,
+          launched_by: Sequelize.TEXT,
+        },
+        { transaction }
+      )
+
+      await queryInterface.createTable(
+        "site_members",
```

Comment:
How would that help?

Reply:
i think it's just for tracking purposes + soft-deletes still preserves data whereas hard delete doesn't. both r fine but i'm just tending towards soft so that we don't lose any data